### ISSUE TITLE
Fix Python support for the Windows port

### ISSUE THREAD

**Issue by TrechNex** [glimpse-editor/Glimpse#178]
Currently the Python console and plug-ins that rely on Python don't work because we build the application without Python support. 

The components are there to run Python scripts, so we need to figure out what we are missing from the current Windows build process documented here: https://wiki.glimpse-editor.org/index.php?title=Building_Glimpse/Windows

**Comment by TrechNex** [comment]
This problem potentially risks delaying the Windows release. I will continue investigating how upstream resolved this problem and keep trying to figure out how to make the Python dependencies available at build time.

**Comment by TrechNex** [comment]
Pushed out to 0.2.0 as this is apparently an upstream issue we have no control over. We should continue to monitor the situation.

**Comment by TrechNex** [comment]
This is now being tracked upstream with a bug I raised here: https://gitlab.gnome.org/GNOME/gimp/issues/4286

**Comment by TrechNex** [comment]
This will be pushed out to beta testing phase as I think this is a build problem rather than a packaging one.

If changes are needed to the WiX Toolset manifest to include extra components then it may delay release, but we'll see how things go.

**Comment by TrechNex** [comment]
I have sunk dozens of hours into trying to figure this out, and it's defeated me! The documented methods from upstream are all poorly-tested and slightly broken because they struggle to find Windows developers to work on them.

If you're the kind of person that both understands how Arch Linux works and have a spare Windows VM knocking around then you will probably have more luck with this than I did!

**Comment by TrechNex** [comment]
For reference:
https://wiki.glimpse-editor.org/index.php?title=Building_Glimpse
https://wiki.gimp.org/wiki/Hacking:Building
https://gitlab.gnome.org/GNOME/gimp/pipelines

Their automated build steps are in the `.gitlab-ci` files in the GNU IMP, BABL and GEGL repositories on that gitlab server.

**Comment by TrechNex** [comment]
I've brought this back in for me to work on during the beta testing phase, as I have a couple of new ideas for how to make this happen.

**Comment by TrechNex** [comment]
Updated the upstream bug ticket with my current progress, as I think we're going to need an understanding of how upstream do their cross-compilation to proceed: https://gitlab.gnome.org/GNOME/gimp/issues/4286#note_722539

I'm going to *try* to get the MinGW-W64 instructions to work. Hopefully third time's the charm!

**Comment by TrechNex** [comment]
Kicked out as a 0.2.x candidate, but will bring back in if I find time to do MinGW-w64 the weekend before we release 0.1.2

**Comment by TrechNex** [comment]
I have this weekend to try and fix it in time for 0.1.2 release. If it can't be done, it'll be kicked out to 0.2.x candidate as we'd be waiting on more information from upstream.

**Comment by TrechNex** [comment]
I make no guarantees that this will be fixed for the 0.1.4 release as it's a "known upstream issue", but it's in the mix just in case.

**Comment by TrechNex** [comment]
There is a ray of hope, as I've just found the cross-build instructions that upstream are using here: https://gitlab.gnome.org/GNOME/gimp/-/blob/master/build/windows/crossbuild-gitlab-ci/build-deps.sh

I will need to check if those are also present in the gimp2-10 branch. If not, then I may need to amend them.

**Comment by TrechNex** [comment]
This may also be of interest: https://gitlab.gnome.org/GNOME/gimp/-/tree/gimp-2-10/build%2Fwindows%2Fjhbuild

**Comment by TrechNex** [comment]
Brought back in to 0.1.4 because we're having to completely re-do our Windows build process to make uplifting BABL and GEGL versions built with `meson` work.

It is likely that we will resolve the missing Python build step at the same time.

**Comment by TrechNex** [comment]
We're closing this as a won't fix because it relies on Python 2, which is now end of life.
The bug can be revisited if we ever rebase on GNU Image Manipulation Program 3.0, as we will have to revamp things anyway to support Python 3.

I will add a to-do item to the new installer fixes to remove Python 2 dependencies we don't need.
